src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -56,4 +59,4 @@ fragment BACKTICK: '`';
 QUOTED_IDENT: BACKTICK (('``') | ~('`'))+ BACKTICK;
 
 fragment DIGIT: [0-9];
-WS  :   [ \t\n\r]+ -> skip ;
+WS  :   [ \t\n\r]+ -> channel(HIDDEN) ;
```

Comment:
oh I see, was expecting it to ouput a new stream that you could re-instantiate the parser with, but the rewrite could've added new tokens and lexer rules, makes sense (kinda hard finding complete examples with TokenStreamRewrite and parser)